### PR TITLE
Species whitelist

### PR DESCRIPTION
**Pull request by Poojawa**
Whitelisting on hold, but doing some tweaks to sprites to make them a bit better sorted on layering

Greyscaled Red Panda tail, Hawk tail. Added Otie ears/tails and shepherd tails.

Rolled species up into Mammal, avian, aquatic, insect. All it really does it slightly alter each sprite, avians get their special foots, insects have the moth sprites. All are greyscaled as well, so they should be colorable.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/furrypeople.dm [hunk 1]
```diff
@@ -15,105 +15,57 @@ datum/species/canid
...
-	mutant_bodyparts = list("beak", "wings", "avian_tail", "taur")
-	default_features = list("beak" = "None", "wings" = "None", "taur" = "None")
+	mutant_bodyparts = list("snout", "wings", "taur", "mam_tail", "mam_body_markings")
+	default_features = list("snout" = "Sharp", "wings" = "None", "taur" = "None", "mam_body_markings" = "Hawk")
 	attack_verb = "peck"
 	attack_sound = 'sound/weapons/slash.ogg'
 	miss_sound = 'sound/weapons/slashmiss.ogg'
-	roundstart = 0 //no sprites yet
-	blacklisted = 1
+	roundstart = 1 //no sprites yet
```
Comment (TalkingCactus) [inline code/modules/mob/living/carbon/human/species_types/furrypeople.dm]: might wanna remove this comment

**Review by TalkingCactus — APPROVED** [review]
looks good, just needs to be tested now

File: code/modules/mob/living/carbon/human/species_types/furrypeople.dm (hunk 1 above)
Comment (Poojawa) [reply code/modules/mob/living/carbon/human/species_types/furrypeople.dm]: Yee

**Comment by Poojawa** [comment]
Confirmed working on my end, they look not-shit too. hooray.

**Pull request merged**